pyproject.toml

```diff
@@ -78,22 +80,18 @@ ruff = "^0.3.4"
 optional = true
 
 [tool.poetry.group.docs.dependencies]
-mkdocs = "*"
-sphinx = "~7.1"
-sphinx-autobuild = "^2021.3.14"
-sphinx-autoapi = "^3.0.0"
+mkdocs = "^1.6.0"
 ipython = "^8.12"
-groundwork-sphinx-theme = "^1.1.1"
-sphinx-copybutton = "^0.5.2"
 nbsphinx = "^0.9.3"
-m2r2 = "^0.3.3.post2"
-furo = "^2023.9.10"
 mock = "^5.1.0"
-autodoc = "^0.5.0"
 dunamai = "^1.19.0"
+mkdocs-material = "^9.5.19"
+mkdocstrings = {extras = ["python"], version = "^0.23.0"}
+mkdocs-include-markdown-plugin = "^6.0.5"
+mkdocs-jupyter = "^0.24.7"
```

Comment:
Would it make sense to also include a `jupyter` dependency for editing / maintaining the `.ipynb` file(s) for this project (I interpreted this dependency as being able to read/integrate the docs but not create or modify the files)?